java/jsinterop/generator/closure/BUILD

```diff
@@ -11,6 +11,7 @@ licenses(["notice"])
 java_library(
     name = "closure",
     srcs = glob(["*.java"]),
+    javacopts = ["-Xep:BetaApi:OFF"],
```

Comment:
Why do you want to do that?
Furthermore, It doesn't seems related to the change.

Reply:
google_bazel_common enables the BetaApi checker by default.
BTW, we could also pass the javacopts from a bazelrc for the project (don't remember the exact name, and on mobile so hard to check; hopefully you'll understand what I mean here)

Reply:
But i should not use any beta guava api in jsinterop generator. Do I?

Reply:
It's probably fine to turn this off - you're not an API, so any changes that you make won't effect your downstream users. Plus, we'll migrate you internally from any beta APIs.